### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iennae**
Resolves Issue #17.

### CODE REVIEW

**Review by lattwood — CHANGES_REQUESTED** [review]
Referring to another file is code

File: README.md [hunk 1]
```diff
@@ -50,4 +50,4 @@ I know right?
 
 ## Contributing
 
-You don't.
+Refer to [Contributing](https://github.com/kelseyhightower/nocode/blob/master/CONTRIBUTING.md) for the most up-to-date process.
```
Comment (marcosalles) [inline README.md]: is code, please remove

**Review by marcosalles — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (got991) [reply README.md:53]: LOL

Comment (Aether-254) [reply README.md:53]: > is markdown code, please remove

LOL.

**Pull request closed without merging**